src/shared/utils/challenge-listing/sort.js

```diff
@@ -34,7 +34,18 @@ export default {
     name: '# of submissions',
   },
   [SORTS.PHASE_END_TIME]: {
-    func: (a, b) => a.currentPhaseRemainingTime - b.currentPhaseRemainingTime,
+    func: (a, b) => {
```

Comment:
I don't competely follow, why do you modify this sort function? I believe, the original code was correct, wasn't it? It trully compared the time till the phase end (which could be any phase, not only checkpoint and submission). The only thing that wasn't good was the measleading name of this sort.

On the other hand, the code itself seems fine logically, just had to be put into [SORTS.TIME_TO_SUBMIT]

Reply:
@birdofpreyru This was the main source of the bug. The original sort here was actually not sorting anything and returned the results unchanged from the server, because the "currentPhaseRemainingTime" is a key from the V2 API which is no longer in use for grabbing challenge listings and is not present in the V3 API. None of the challenge results contain this key anymore. This SORT.PHASE_END_TIME was the sort that was being used for "time to submit" in open for registration listings, was labeled as such and was used nowhere else, which is why I modified this one.